src/modules/seq2seq_decoder.py

```diff
@@ -23,6 +23,7 @@
 
 from .modules import Pooler
 
+import warnings
```

Comment:
Could you switch to log.warning? It's what we use elsewhere, and it should work more straightforwardly with our logging tools. Example:

https://github.com/nyu-mll/jiant/blob/master/main.py#L178

Reply:
fixed!